### DISCUSSION TITLE
可否在同一个对象中注入同名的实例函数和静态函数

### DISCUSSION THREAD

**Discussion by yqs112358** [Tencent/ScriptX#26] [Q&A] [answered]
不知是否可以在同一个类中导入两个函数，一个function一个instanceFunction，但两者导入在脚本引擎里的名字相同
形如
.....
.instanceFunction("read",&Class::readFile)
.function("read",&Class::read)
.build()

@LanderlYoung

**Comment by LanderlYoung** [comment, marked as answer]
不建议这么做。

具体要看backend实现，现在JS是把instance挂到prototype里，lua是metaTavle里。
